en/physics-2d/physics-2d-system.md

```diff
@@ -0,0 +1,158 @@
+# Physics Manager
+
+The physical system hides most of the implementation details of the physical modules (Box2D and Builtin modules) (e.g. creating rigid bodies, synchronizing rigid body information to nodes, etc.).
+
+You can use the physical system to access some of the common functions of the physics module, such as click testing, ray testing, and setting up test messages.
```

Comment:
You can use the physical system to access some of the common functions of the physics module, such as click testing, ray testing, and setting up test messages.
-->
Use the physics system to access some of the common functions of the physics module, such as click testing, ray testing, and setting up test messages.